package.json

```diff
@@ -7,6 +7,7 @@
     "generate-function": "^2.0.0"
   },
   "devDependencies": {
+    "pre-commit": "^1.2.2",
```

Comment:
what does this do? :)

Reply:
Add a git hook in order to run `npm test` before commit. That force the developer to create "non-broken" commits https://www.npmjs.com/package/pre-commit

Anyway I'm not forcing anyone. If you want I'll remove it

Reply:
Ya I'd prefer not to include this for now.

Reply:
Commit removed